### PR TITLE
Remove need for schema creation permissions when deploying to PaaS

### PR DESCRIPTION
**Pull request by samcrang**

### CODE REVIEW

File: src/main/resources/paas.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ externalConfigDirectory: /tmp
 
 downloadConfigs: true
 
-fieldsYamlLocation: classpath://config/fields.yaml
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: https://field.discovery.openregister.org/records.yaml
```
Comment (karlbaker02) [inline src/main/resources/paas.yaml]: I'm guessing you didn't mean to include these changes here?

Comment (samcrang) [inline src/main/resources/paas.yaml]: I did. Do you not feel it's the right thing to do?

Comment (samcrang) [inline src/main/resources/paas.yaml]: Actually you're right—this should probably be configurable in `manifest.yaml` but with a sensible default.

Comment (karlbaker02) [inline src/main/resources/paas.yaml]: I was thinking more along the lines that we allow the phase to be specified (via register domain right?), so this probably shouldn't be pointing directly at discovery.

Comment (samcrang) [inline src/main/resources/paas.yaml]: Or just make it possible to set anything. Then you could have your own register/field registers deployed on PaaS.

**Comment by samcrang** [comment]
How's this @karlbaker02?

**Pull request merged**